pkg/controller/data_upload_controller.go

```diff
@@ -343,7 +343,12 @@ func (r *DataUploadReconciler) runCancelableDataUpload(ctx context.Context, fsBa
 	tags := map[string]string{
 		velerov1api.AsyncOperationIDLabel: du.Labels[velerov1api.AsyncOperationIDLabel],
 	}
-	if err := fsBackup.StartBackup(path, fmt.Sprintf("%s/%s", du.Spec.SourceNamespace, du.Spec.SourcePVC), "", false, tags, du.Spec.UploaderConfig); err != nil {
+
+	if du.Spec.DataMoverConfig == nil {
+		du.Spec.DataMoverConfig = &map[string]string{}
+	}
+
+	if err := fsBackup.StartBackup(path, fmt.Sprintf("%s/%s", du.Spec.SourceNamespace, du.Spec.SourcePVC), "", false, tags, *du.Spec.DataMoverConfig); err != nil {
```

Comment:
Let's make the parameter of `StartBackup` as *map[string]string, so that we don't need to initialize `du.Spec.DataMoverConfig`